app/pages/HomePage/components/HomePageBackground/HomePageBackground.scss

```diff
@@ -0,0 +1,14 @@
+.homePageBackground {
+  width: 100vw;
+  height: 500px;
+  background-image: url(./assets/hero.jpg);
+  background-size: cover;
+  background-position: center;
+  position: absolute;
+  top: 70px;
+  right: 0;
+  z-index: 1;
```

Comment:
I've always been hesitant to rely on z-index too much, since I've found that it can easily lead to tight coupling between otherwise independent elements on a page. I especially feel this way if I ever have to carefully set the z-index of multiple different elements to ensure that they stack properly.

I think you may be able to accomplish what you want aesthetically with the background that fades into the next section using a `::before` pseudoelement. I can create a PR against this branch to demonstrate that.

If I can't persuade you to not absolutely position this element, would you at least be interested in setting the z-index here to `-1`? I think that should allow you to not have to set the z-index to 2 or 3 in the other CSS styles that you touched in this PR.